mendix/widgets-resources
# Data grid - Fixes & Improvements

In this PR I am:
- Fixing structure preview of data grid filters when adjustable by user is false [WC-249]
- Fixing height of filters in structure preview [WC-248]
- Fixing hidden columns not being rendered in Studio [WC-245]
- Adding different color in structure preview for hidden columns [WC-211]
- Forcing caption to be required if column is hidden by default [WC-244]
- Defining filters to be enabled by default
- Renaming empty placeholder dropzone in Structure preview
- Changing the advanced toggle to be inside configurations for Studio Pro and as last group in Studio

How to test?
- Run a project using the latest module or copy from Data Grid project (SVN)
- Everything should be working, including fix for issues related on top and the new colors.
